services/sql-migrator/data-dev.go

```diff
@@ -0,0 +1,7 @@
+// +build dev
+
+package migrator
+
+import "net/http"
+
+var MigrationAssets http.FileSystem = http.Dir("./sql/migrations")
```

Comment:
Why do you need a separate file for this? Is it okay to move it to migrator.go?

Reply:
This is only to by run in development mode, using "dev" build tag, in order to assist development. It will not be part of the final rudder-server binary.